AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -113,10 +113,20 @@ export class SummaryComponent implements OnInit, OnDestroy {
         this.bookinConfirmed = this.hearing.status === 'Created';
     }
 
+    setAudioRecordingRequired(currentValue: boolean): boolean {
+        // CACD hearings should always have recordings set to off
+        if (this.caseType === 'Court of Appeal Criminal Division') {
```

Comment:
just to aid future re-use, could we maybe move this case type to `constants.ts`?

something like:

`CaseTypes: { CourtOfAppealCriminalDivision: 'Court of Appeal Criminal Division' }`

we could also reference this from the associated test as well.

Reply:
I have now made this change and added the reference constants.ts